extensions/applicationinsights-clickanalytics-js/src/DataCollector.ts

```diff
@@ -0,0 +1,173 @@
+/**
+ * @copyright Microsoft 2020
+ */
+
+import {
+    getLocation, getDocument, getWindow, hasDocument
+} from '@microsoft/applicationinsights-core-js';
+import { _findClosestAnchor, isValueAssigned } from './common/Utils';
+import { IClickAnalyticsConfiguration, IOverrideValues } from './Interfaces/Datamodel';
+
+
+var clickCaptureInputTypes = { BUTTON: true, CHECKBOX: true, RADIO: true, RESET: true, SUBMIT: true };
+
+
+/**
+ * Get Image href of a given HTMLImageElement
+ * @param element - An html image element
+ * @returns Href value.
+ */
+export function _getImageHref(element: HTMLImageElement): string {
```

Comment:
Nit: As these functions are exported they shouldn't start with an underscore (as thats for private / protected functions / properties)